recipes/libqasm/all/conanfile.py

```diff
@@ -92,7 +96,6 @@ def generate(self):
         deps = CMakeDeps(self)
         deps.generate()
         tc = CMakeToolchain(self)
-        tc.variables["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
CMakeToolchain passes BUILD_SHARED_LIBS always, so you dont need to care about.

Reply:
Good to know!